src/zif_abapgit_definitions.intf.abap

```diff
@@ -364,6 +364,20 @@ INTERFACE zif_abapgit_definitions
     tty_col_spec TYPE STANDARD TABLE OF ty_col_spec
                       WITH NON-UNIQUE KEY tech_name.
 
+  TYPES: tt_langu TYPE STANDARD TABLE OF langu.
```

Comment:
is it possible to place these definitions somewhere else? this interface is too messy already

Reply:
I can pass to zcl_abapgit_objects_super and made a change to read all languages supported, this way we have always (for all object types) system languages

Reply:
I'd like not to change `zcl_abapgit_objects_super`, there is a parallel task looking into overall translation approach

Reply:
What about ZCL_ABAPGIT_OBJECTS_PROGRAM?
Objective of this is always have available languages

Reply:
Created new interface zif_abapgit_lang_definitions